ws-android/files/workspace/index.html

```diff
@@ -0,0 +1,6 @@
+<h1 align=center> Your site to download your App to your phone.</h1>
+Note: on a free cloud9 site <a href="https://c9.io">https://c9.io</a> this web page needs to be restarted every 24 hours
```

Comment:
Why does it need to be restarted? Let's fix the problem instead of offloading to end users.